static/sass/_pattern_footer.scss

```diff
@@ -1,44 +1,15 @@
 @mixin canonical-p-footer {
-  .p-footer {
-    @extend %vf-strip;
+  .row:first-child{
+    position: relative;
 
-    background-image: linear-gradient(
-        to bottom left,
-        rgba(119 41 83 / 100%) 0%,
-        rgba(119 41 83 / 100%) 49.9%,
-        rgba(119 41 83 / 0%) 50%,
-        rgba(119 41 83 / 0%) 100%
-      ),
-      linear-gradient(
-        to bottom left,
-        $color-accent-background 0%,
-        $color-accent-background 100%
-      );
-    background-position: top right;
-    background-repeat: no-repeat;
-    background-size: 25% 15%, 100%;
-    clip-path: polygon(100% 0%, 100% 100%, 0% 100%, 0% calc(0% + 4rem));
-    color: $color-mid-light;
-    padding-bottom: 2.5rem;
-    padding-top: 6rem;
-
-    &__link {
-      color: $color-mid-light;
-
-      &:hover,
-      &:visited {
-        color: $color-mid-light;
-      }
-    }
-
-    hr {
-      background: rgba(255 255 255 / 25%);
-      margin: 2rem 0;
+    .p-inline-list{
```

Comment:
This will also affect every `p-inline-list` on the site. I would suggest if you are pushing something down you could create a new utility for this so its clear to others what is going on.

Reply:
Added the utility, lmk if this is what you meant!